src/lib/php/Dao/LicenseDao.php

```diff
@@ -351,7 +351,8 @@ public function getLicensesPerFileNameForAgentId(ItemTreeBounds $itemTreeBounds,
     $rgtStack = array($row['rgt']);
     $lastLft = $row['lft'];
     $path = implode('/', $pathStack);
-    $this->addToLicensesPerFileName($licensesPerFileName, $path, $row, $ignore, $clearingDecisionsForLicList);
+    $uploadTreeId = $row['uploadtree_pk'];
```

Comment:
As discussed please remove uploadtreeid from here and use itemtreebonds

Reply:
I am on it. Did some testing of the function last night. Moving on to implementation

Reply:
resolved